packages/charts/src/specs/settings.tsx

```diff
@@ -417,6 +417,7 @@ export interface LegendSpec {
   /**
    * Limit the legend to the specified maximal depth when showing a hierarchical legend
    */
+  //TODO this is not the max depth, but the number of level shown: 0 none, 1 first, 2 up to the second etc
   legendMaxDepth: number;
```

Comment:
```suggestion
  /**
   * Limit the legend to the specified maximal depth when showing a hierarchical legend
   * 
   * TODO: this is not the max depth, but the number of level shown: 0 none, 1 first, 2 up to the second etc
   */
  legendMaxDepth: number;
```

Can you put this comment in the TDDoc comment?

Reply:
done in https://github.com/elastic/elastic-charts/pull/1956/commits/18913e0ccbb8549d45a624502e59fa854c76e116